posthog/settings/web.py

```diff
@@ -50,24 +54,35 @@
     # ok below the above middlewares however.
     "posthog.health.healthcheck_middleware",
     "posthog.middleware.ShortCircuitMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
```

Comment:
Do we need the session middleware? I assume this would produce a DB lookup if the user sends a cookie.

Reply:
I added this because django wouldn't start without it while running locally (again, I was looking at this from a pods perspective, rather than the specific endpoint)